homeassistant/components/recorder/statistics.py

```diff
@@ -129,12 +130,12 @@
 
 
 STATISTIC_UNIT_TO_UNIT_CONVERTER: dict[str | None, type[BaseUnitConverter]] = {
+    **{unit: AreaConverter for unit in AreaConverter.VALID_UNITS},
     **{
         unit: BloodGlugoseConcentrationConverter
         for unit in BloodGlugoseConcentrationConverter.VALID_UNITS
     },
     **{unit: ConductivityConverter for unit in ConductivityConverter.VALID_UNITS},
-    **{unit: ConductivityConverter for unit in ConductivityConverter.VALID_UNITS},
```

Comment:
This fixes a mistake introduced when adding the blood glucose units, we should fix it in a separate PR not in this one: https://github.com/home-assistant/core/pull/130375

(This PR just needs to be updated/rebased after merge of 130375)

Reply:
Done